indigo_app/static/stylesheets/_akn.scss

```diff
@@ -1,99 +0,0 @@
-/* akoma ntoso styles */
-@import 'lib/indigo-web/scss/indigo-web';
-
-/* overrides for indigo inserts into akoma-ntoso trees */
```

Comment:
We need to keep all of these, please. They're necessary for some of the decorations that Indigo adds on top of indigo-web.